libs/types/src/lib/supabase.gen.ts

```diff
@@ -77,6 +77,7 @@ export interface Database {
           last_name: string;
           role: number | null;
           user_id: string;
+          application_status: number;
```

Comment:
instead of modifying it manually, do `yarn supabase:gen-types` on your local setup to check instead